### PR TITLE
Docs for bumping an experimental module's version

### PR DESCRIPTION
**Pull request by codebien**
It adds documentation about the steps for upgrading experimental modules.

### CODE REVIEW

File: js/modules/k6/experimental/README.md [hunk 1]
```diff
@@ -8,3 +8,14 @@ Although [accessible in k6 scripts](../../../initcontext.go) under the `k6/exper
 * [`k6/experimental/k6-timers`](https://github.com/grafana/xk6-timers)
 
 While we intend to keep these modules as stable as possible, we may need to add features or introduce breaking changes. This could happen at any time until we release the module as stable. **use them at your own risk**.
+
+## Upgrade version
+
+Experimental modules are based on xk6-extensions, and they introduce a cycle dependency between k6 and the extension. Upgrading an extension's version is required to run the following steps:
```
Comment (na--) [inline js/modules/k6/experimental/README.md:14]:
```suggestion
Experimental modules are based on xk6-extensions, and they introduce a cycle dependency between k6 and the extension. When upgrading an extension's version, it's required to run the following steps:
```

**Review by na-- — DISMISSED** [review]
LGTM besides the minor nitpick

**Review by na-- — DISMISSED** [review]

File: js/modules/k6/experimental/README.md [hunk 2]
```diff
@@ -8,3 +8,14 @@ Although [accessible in k6 scripts](../../../initcontext.go) under the `k6/exper
 * [`k6/experimental/k6-timers`](https://github.com/grafana/xk6-timers)
 
 While we intend to keep these modules as stable as possible, we may need to add features or introduce breaking changes. This could happen at any time until we release the module as stable. **use them at your own risk**.
+
+## Upgrade version
```
Comment (olegbespalov) [inline js/modules/k6/experimental/README.md:12]:
```suggestion
## Upgrading experimental modules
```

**Review by na-- — APPROVED** [review]

**Review by olegbespalov — APPROVED** [review]
👍

**Pull request merged**